### PR TITLE
feat: updated publish script and action YAML

### PR DESCRIPTION
**Pull request by jimezesinachi**
# SynthQL pull request template

## Why

We want new minor versions of each package to be published on every merge into `master`

## What changed

The publish script and the cd.yml action file

## Important

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -37,12 +40,12 @@ jobs:
               run: yarn ci
 
             - name: Set Git user name
-              run: git config --global user.name "Fernando Hurtado"
+              run: git config user.name "github-actions[bot]"
 
             - name: Set Git user email
-              run: git config --global user.email "fernandohur@gmail.com"
+              run: git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```
Comment (fhur) [inline .github/workflows/cd.yml:46]: Where is this ID/email coming from? Can you add a bit of documentation to the yaml file explaining?

File: packages/backend/src/execution/executors/QueryProviderExecutor/convertWhereToQueryProviderInput.test.ts [hunk 2]
```diff
@@ -4,6 +4,8 @@ import { DB } from '../../../tests/generated';
 import { Table, col } from '@synthql/queries';
 
 describe('convertWhereToQueryProviderInput', () => {
+    const date = new Date();
```
Comment (fhur) [inline packages/backend/src/execution/executors/QueryProviderExecutor/convertWhereToQueryProviderInput.test.ts:7]: good catch!

**Review by fhur — CHANGES_REQUESTED** [review]
Overall looks OK to me. Just curious about the weird email address.

**Comment by jimezesinachi** [comment]
> Overall looks OK to me. Just curious about the weird email address.

It's the GitHub Actions 'bot' email address. It's specifically created by GitHub for this kind of use. If you look at the commits created with it on this PR, it has the correct profile picture.

**Review by fhur — APPROVED** [review]

**Pull request merged**